.travis.yml

```diff
@@ -1,11 +1,16 @@
-language: python
+language: c
```

Comment:
Maybe just add a note about that in a comment?  I know I'll forget why this is what it is by the next time I see it :)